common/queue/src/main/java/org/thingsboard/server/queue/settings/TbRuleEngineQueueAckStrategyConfiguration.java

```diff
@@ -24,5 +24,7 @@
     private int retries;
     private double failurePercentage;
     private long pauseBetweenRetries;
+    private boolean multiplyPauseBetweenRetries;
```

Comment:
Maybe we don't need this flag at all? Just check if `max-pause-between-retries > pause-between-retries` to calculate this.

Reply:
@ashvayka I guess that someone may be no need to increase the pause, in some cases, so that's why I think that we need to add this flag.